### DISCUSSION TITLE
[nice-to-have] Clickable part / column of a row in Collection's views.List

### DISCUSSION THREAD

**Discussion by Ontopic** [payloadcms/payload#210] [Feature Requests & Ideas]
Currently the first column is made clickable, the rest of the columns are not. The setup with `useAsTitle` and `defaultColumns` gets everything I would want to change sorted in a nice way in most cases, but when I want to, f.e., make the first column a thumbnail or flag it feels weird I can not click on the `title` or `slug` columns. Or when a variable that may or may not exist in a certain Document set as the first in `defaultColumns` may even result in unreachable (for non-devs) rows, since it's just an empty `Col`

All of it avoidable right now, so no rush, some more control would be nice though without breaking into custom componentas. Perhaps worth considering making more columns clickable by default, or include a boring edit icon that always appears. Would hate that for the clean look though, but maybe a check on if a field is nillish and show a fallback?

**Comment by Ontopic** [comment]
I understand emphasising the first column as clickable with underline link-like styling and keeping the rest of the row clean from a design perspective. I can however not yet think of a reason to not just make the whole row clickable (without changing styling necessarily, except for `cursor: pointer`). If people want custom dropdown actions from the list view (something on my todo list to try out soon 🤗) they'll have to resort to custom cells anyway and can / will automatically override any "aggressive" event capturing

**Comment by jmikrut** [comment]
Hey @Ontopic — I think you're totally right here. It's more sensible to have the whole row be clickable and would not be too difficult to adapt to. 

We'll slate this in for a future feature (will technically be a breaking change necessitating a minor release). We are working on our Roadmap as well as some other pretty exciting things which will take priority here but nonetheless, this will happen at some point soon!

**Comment by taismassaro** [comment]
hey @jmikrut, are there any updates on this?
